src/pages/NftCollections/List.tsx

```diff
@@ -113,6 +114,23 @@ const HolderMinterSort = () => {
     </div>
   )
 }
+interface SimpleSortProps {
+  sortField: 'transactions' | 'timestamp'
+  fieldI18n: string
+}
+const SimpleSortHeader: React.FC<SimpleSortProps> = ({ sortField, fieldI18n }) => {
+  const sortParam = useNFTCollectionsSortParam()
+  const { handleSortClick } = sortParam
+  return (
+    <span>
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
+      <span onClick={() => handleSortClick(sortField)} role="button" tabIndex={0}>
```

Comment:
I would suggest using a link leading to `/xxx?sorted_by=xxx`, e.g. `/nft-collection?sorted_by=created_time.asc`. By doing so, the ordered list can be shared by URL.

Reply:
But don't miss other url queries in the url

Reply:
I found an existing hook for `sort` at https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/hooks/route.ts#L22

Reply:
I think this feature is support now. When I click the sort, the url change to  `http://localhost:3000/en/nft-collections?sort=timestamp.desc`